Jenkinsfile

```
@@ -1 +1,5 @@
-buildPlugin(timeout: 120)
+buildPlugin(timeout: 120, useAci: true, platforms: [
```

Oops, forgot to fix locally:

```suggestion
buildPlugin(timeout: 120, useAci: true, configurations: [
```